src/main.py

```diff
@@ -161,16 +167,46 @@ def open_pr(subject, body, branch):
     base = "master"
     head = "{}:{}".format(repo.owner.login, branch)
     pr_message = ((body or "") + "\n\n" + DISCLAIMER).strip()
-    # Include closed PRs – if the maintainer has closed our last PR, we don't want to
-    # open another one.
-    for pr in origin_repo.get_pulls(state="all", base=base, head=head):
+
+    if os.path.isfile("flathub.json"):
+        with open("flathub.json") as f:
+            repocfg = json.load(f)
+            automerge = repocfg.get("automerge-flathubbot-prs")
+
+    # Enable automatic merge if external data is broken unless explicitly disabled
+    if automerge is not False and manifest_checker:
```

Comment:
```suggestion
    if automerge and manifest_checker:
```

Reply:
That would make no sense. It's supposed to enable automerge if it's not explicitly disabled (thus `is not False`) in flathub.json and some source is broken.